lua-api-crates/filesystem/src/path.rs

```diff
@@ -286,7 +278,39 @@ impl UserData for Path {
                 .globals()
                 .get::<_, mlua::Table>("string")?
                 .get::<_, mlua::Function>("len")?;
-            len.call::<_, LuaString>(lua_str)
+            len.call::<_, mlua::Integer>(lua_str)
+        });
+        methods.add_method("format", |lua: &'lua Lua, this, multi: LuaMultiValue| {
+            let lua_str = path_to_lua_str(lua, &this)?;
+            let format = lua
+                .globals()
+                .get::<_, mlua::Table>("string")?
+                .get::<_, mlua::Function>("format")?;
+            format.call::<_, LuaMultiValue>((lua_str, multi))
+        });
+        methods.add_method("byte", |lua: &'lua Lua, this, multi: LuaMultiValue| {
+            let lua_str = path_to_lua_str(lua, &this)?;
+            let format = lua
+                .globals()
+                .get::<_, mlua::Table>("string")?
+                .get::<_, mlua::Function>("format")?;
```

Comment:
Shouldn’t this be byte?

Reply:
Yes, I had a few of these copying mistakes. I think it should be fixed now.